packages/grid/_modules_/grid/constants/eventsConstants.ts

```diff
@@ -1,85 +1,431 @@
 // Web standard events
+
+/**
+ * Fired when the grid is resized. Called with a [[GridResizeParams]] object.
+ * @event
+ */
 export const GRID_RESIZE = 'resize';
+
+/**
+ * Fired when a `focusout` event happens in the grid.
+ * @event
+ */
 export const GRID_FOCUS_OUT = 'focusout';
+
+/**
+ * Fired when a `keydown` event happens in the grid.
+ * @event
+ */
 export const GRID_KEYDOWN = 'keydown';
+
+/**
+ * Fired when a `keyup` event happens in the grid.
+ * @event
+ */
 export const GRID_KEYUP = 'keyup';
+
+/**
+ * @ignore - do not document.
+ */
 export const GRID_NATIVE_SCROLL = 'scroll';
```

Comment:
It's never published.

Reply:
Dead code to remove then?

Reply:
The constant is only used here: https://github.com/mui-org/material-ui-x/blob/master/packages/grid/_modules_/grid/hooks/features/virtualization/useGridVirtualRows.ts#L411. Maybe it should be a local variable.